text/0000-deprecate-string-based-actions.md

```diff
@@ -0,0 +1,89 @@
+- Start Date: 2020-05-23
+- Relevant Team(s): Ember.js
+- RFC PR: (after opening the RFC PR, update this with a link to it and update the file name)
+- Tracking: (leave this empty)
+
+# Deprecate String Based Actions
+
+## Motivation
+
+> In Ember 1.13 we introduced [closure actions](https://github.com/emberjs/rfcs/blob/00ac2685c86f27d41547012903f485a4ef338d27/active/0000-improved-actions.md) have been recommended over string based actions. This officially deprecates string based actions in favor of closure actions,
+which provide many benefits such as
+
+1. simpler passing through components
+2. less need for an actions hash
+3. ability to return a value
+4. typing in TypeScript
+5. compatible with Glimmer Components
+
+## Detailed Design
```

Comment:
This is missing quite a bit of info I think. 

* Which framework classes are affected?
* What are the replacement values for each of the various use cases for `.send`?
* Shouldn't it also deprecate directly accessing `this.actions` (this is a thing some folks have done, and the `@action` decorator still populates this actions hash)?
* Does it deprecate specifying an `actions` hash?
* What is the exact prose that will be included in the deprecation guide entry?

Reply:
> Shouldn't it also deprecate directly accessing this.actions (this is a thing some folks have done, and the @action decorator still populates this actions hash)?

Added to unresolved questions for the moment.

> Does it deprecate specifying an actions hash?

Added to unresolved questions for the moment.

Reply:
> Which framework classes are affected?

Added.

> What are the replacement values for each of the various use cases for .send?

Partially added.

> What is the exact prose that will be included in the deprecation guide entry?

Help wanted here.